config/playground/eks/dev/README.md

```diff
@@ -0,0 +1,52 @@
+# Terraform - Elastic Kubernetes Getting Started Guide
+
+## Documentation
```

Comment:
move reference to the appendix part.

but add a summary about what do we do in this doc.

Reply:
Fixed. Thanks